src/components/DisplayArticlesList.jsx

```diff
@@ -1,31 +1,29 @@
-import React, { useState, useEffect } from "react";
+import React, { useEffect } from "react";
 import { getArticles } from "../modules/getArticles";
 import ArticleCard from "./ArticleCard";
 import { Card, Container } from "semantic-ui-react";
+import { useDispatch, useSelector } from "react-redux";
 
 const DisplayArticlesList = () => {
-  const [articleData, setArticleData] = useState([]);
-  const getArticleData = async () => {
-    let response = await getArticles.index();
-    setArticleData(response);
-  };
+  const dispatch = useDispatch();
+  const articleList = useSelector((state) => state.newsFeed);
```

Comment:
A small detail, but you can omit the parenthesis if you only have one argument. That goes for all your functions. And again, I see no reason to change the name. I would go with:
```js
const { newsFeed }  = useSelector(state => state);
```

Reply:
Fixed